src/consts/warpRoutes.ts

```diff
@@ -1,9 +1,182 @@
-import { WarpCoreConfig } from '@hyperlane-xyz/sdk';
+import { TokenStandard, WarpCoreConfig } from '@hyperlane-xyz/sdk';
+import { zeroAddress } from 'viem';
+
+const ROUTER = '0xe0c8f83bA0686FDF1a76AF0cC202181AEaA25a03';
+const ITT = '0x5f94BC7Fb4A2779fef010F96b496cD36A909E818';
 
 // A list of Warp Route token configs
 // These configs will be merged with the warp routes in the configured registry
 // The input here is typically the output of the Hyperlane CLI warp deploy command
 export const warpRouteConfigs: WarpCoreConfig = {
-  tokens: [],
+  tokens: [
+    {
+      addressOrDenom: ITT,
```

Comment:
I think in warpcore, `addressOrDenom` is actually the router and `collateralOrDenom` would be the token contract, curious why you had to deviate here

Reply:
Oh, this comes from the initial approach (and the currently running patched intent's framework UI) and how the gas/allowances/balances were calculated.

So, if I set the `ROUTER` in `addressOrDenom` I get the error: ["Connected token not found..."](https://github.com/bootnodedev/hyperlane-monorepo/blob/c40b2c676d02a1d65f99314521f098fb1ddd0f4b/typescript/sdk/src/warp/WarpCore.ts#L108)

And IIUC I shouldn't be setting the `connected` tokens to point to the `ROUTER` address, instead of `ITT`.

I'll re-check docs and code to see if I'm missing something